### PR TITLE
[sui-ssr] Add PRE_SSR_HANDLER hook

### PR DESCRIPTION
**Pull request by carlosvillu**
This change is [Reviewable](https://reviewable.io/reviews/sui-components/sui/658)

### CODE REVIEW

File: packages/sui-ssr/server/index.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ let _memoizedHtmlTemplatesMapping = {}
         customTlds: /localhost|\.local/
       })
 
-      !parsedUrl || parsedUrl.tld === 'localhost' // eslint-disable-line
+        !parsedUrl || parsedUrl.tld === 'localhost' // eslint-disable-line
```
Comment (midudev) [inline packages/sui-ssr/server/index.js]: Is the linter working correctly? Some changes seems to be off.

File: packages/sui-ssr/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/ssr",
-  "version": "5.21.0",
+  "version": "5.21.0-beta.0",
```
Comment (midudev) [inline packages/sui-ssr/package.json]: 👀

File: packages/sui-ssr/hooks-types.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 export default {
-  PRE_HEALTH: 'pre_health',
-  LOGGING: 'logging',
+  APP_CONFIG_SETUP: 'app_config_setup',
   INTERNAL_ERROR: 'internal_error',
+  LOGGING: 'logging',
   NOT_FOUND: 'not_found',
-  APP_CONFIG_SETUP: 'app_config_setup',
+  PRE_HEALTH: 'pre_health',
+  PRE_SSR_HANDLER: 'pre_ssr_handler',
```
Comment (kikoruiz) [inline packages/sui-ssr/hooks-types.js]: OH GOD!! Thanks for sorting this object 🙏

File: packages/sui-ssr/server/index.js (hunk 1 above)
Comment (carlosvillu) [reply packages/sui-ssr/server/index.js]: I dont know, something with the linter. It is prettier.

File: packages/sui-ssr/package.json (hunk 2 above)
Comment (carlosvillu) [reply packages/sui-ssr/package.json]: this will be change for the 5.22 in the release

**Pull request merged**